DocumentationValidator/Program.cs

```diff
@@ -14,11 +15,69 @@ class Program
         static async Task<int> Main(string[] args)
         {
             IList<NuGetLogCode> allLogCodes = GetNuGetLogCodes();
-            List<NuGetLogCode> undocumentedLogCodes = new();
             HttpClient httpClient = new();
+            var cred = args.Length > 0 ? args[0] : "123";
+            var client = new GitHubClient(new ProductHeaderValue("nuget-documentation-validator"))
+            {
+                Credentials = new Credentials(cred)
+            };
 
-            Console.WriteLine($"Checking {allLogCodes.Count} log codes for documentation.");
+            List<NuGetLogCode> undocumentedLogCodes = await GetUndocumentedLogCodes(allLogCodes, httpClient);
+            Console.WriteLine($"Undocumented Log Codes Count {undocumentedLogCodes.Count}");
+
+            Dictionary<NuGetLogCode, List<string>> logCodeToIssue =
+                await GetIssuesForLogCodeAsync(client, undocumentedLogCodes);
+            PrintIssues(logCodeToIssue);
+
+            bool shouldCreateIssues = false;
+            _ = args.Length >= 2 ? bool.TryParse(args[1], out shouldCreateIssues) : false;
+
+            if (shouldCreateIssues)
```

Comment:
Consider a prompt where option 1 is "List Undocumented Log Codes" `shouldCreateIssues=false`, and then option 2 is "Create Issues for Undocumented Log Codes", `shouldCreateIssues=true`.

That is, if you wanted to be able to avoid querying the undocumented issues twice; once to just query; again to create. (my guess for your workflow would be, first you'd want to make sure there's a reasonable number, skim through them, then go back with the flag to automatically create issues?)

Reply:
I think it'd be easier to just add a parser. 

I'll try to spend 10-15 mins on that.